### PR TITLE
Gui tests 3433 5795

### PR DESCRIPTION
**Pull request by maryvictol**
This PR adds testcases and e2e tests for
- #3433
- Node memory limits

### CODE REVIEW

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java [hunk 1]
```diff
@@ -399,6 +405,49 @@ public void forcibleTerminateInstancesIfJobIsStuckInUmount() {
...
+            loginAs(admin);
+            setUserSettings("*");
+            logoutIfNeeded();
+            loginAs(user);
+            tools()
+                    .perform(registry, group, tool, ToolTab::runWithCustomSettings)
+                    .expandTab(EXEC_ENVIRONMENT)
+                    .expandTab(ADVANCED_PANEL)
+                    .setTypeValue("r6i.xlarge")
+                    .setPriceType(ON_DEMAND)
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java:420]: Would we like to add .doNotMountStoragesSelect(true)?

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java [hunk 2]
```diff
@@ -15,7 +15,12 @@
  */
 package com.epam.pipeline.autotests;
 
+import static com.codeborne.selenide.Condition.text;
 import com.epam.pipeline.autotests.ao.PipelineRunFormAO;
+import static com.epam.pipeline.autotests.ao.Primitive.ADVANCED_PANEL;
+import static com.epam.pipeline.autotests.ao.Primitive.EXEC_ENVIRONMENT;
+import static com.epam.pipeline.autotests.ao.Primitive.NODE_IMAGE;
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java]: Clean up the unused import statement, please

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/PlatformPreferencesTest.java [hunk 3]
```diff
@@ -235,6 +237,48 @@ public void allowToSpecifyLustreFSTypeAndThoughput() {
...
+                            .assertPageContainsStringsWithRegex(command, messages)
+                            .close());
+            final LogAO logAO = runsMenu()
+                    .showLog(getLastRunId())
+                    .shouldHaveRunningStatus()
+                    .clickTaskWithName("OOM Logs");
+            final Set<String> logMess = logAO.logMessages().collect(toSet());
+            assertTrue(logMess.stream()
+                    .anyMatch(Pattern.compile(logMessage)
+                            .asPredicate()));
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/PlatformPreferencesTest.java]: Could we add the message which message was not found with run id for convenience?

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/ShellAO.java [hunk 4]
```diff
@@ -109,6 +109,14 @@ public ShellAO assertPageAfterCommandNotContainsStrings(String command, String..
         return this;
     }
 
+    public ShellAO assertPageContainsStringsWithRegex(String command, String ... messages) {
+        String log = lastCommandResult(command);
+        Arrays.stream(messages)
+                .forEach(message ->
+                        assertTrue(Pattern.compile(message).matcher(log).find()));
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/ShellAO.java]: Could we add the message which message was not found with run id for convenience?

**Review by kamyshova — CHANGES_REQUESTED** [review]

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java (hunk 1 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java:420]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java (hunk 2 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/PlatformPreferencesTest.java (hunk 3 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/PlatformPreferencesTest.java]: Done

**Review by NShaforostov — APPROVED** [review]

**Review by kamyshova — APPROVED** [review]

**Pull request merged**